### DISCUSSION TITLE
Importing a second SBOM into the same project appears to have deleted the first

### DISCUSSION THREAD

**Discussion by nigellh** [DependencyTrack/dependency-track#3050] [General]
Hi. A colleague had imported an SBOM with around 4500 entries.

A little later she imported a second smaller SBOM and it appears to have deleted the first and just left 7 entries.

Is this the correct expectation?

Couldn't it give an option to choose?

Thanks.

**Comment by fmarot** [comment]
I'm not an expert but in my opinion this is the expected behavior: you
expect Sbom for a given project and version to be self sufficient. If you
upload another bomb with same name/Id and version, I expect it to overwrite
previous ones.

Regards
